recipes/chemicals/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "chemicals" %}
+{% set version = "0.1.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/chemicals-{{ version }}.tar.gz
+  sha256: db508a38e0e102e83bf1facb40b55ac428ed21fe34f9e342b1b0601312142672
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - fluids >=0.1.86
+    - pandas
+    - python
+    - scipy
+
+test:
+  imports:
+    - chemicals
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/CalebBell/chemicals
+  summary: Chemical properties component of Chemical Engineering Design Library (ChEDL)
```

Comment:
Sorry to bother you again, @CalebBell. Is this summary accurate enough? It looks like another package you have. Please feel free to suggest improvements.